docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,121 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/17/2020
+---
+# Inspect managed stack traces (dotnet-stack)
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
```

Comment:
I think this issue applies to other tools as well so perhaps best for a separate PR. I expect customers to treat "Applies to" as "what .NET runtime version can I diagnose with the tool"
```suggestion
**This article applies to:** ✔️ .NET Core 3.0 and later versions
```

Reply:
This tool should work for 3.0 as well.